ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -609,6 +621,115 @@ function AuthProviderForm({
                             />
                         </FormSection>
                     )}
+                    {selectedAuthProvider.type === 'oidc' && (
+                        <FormSection
+                            title="Claim mappings for the authentication provider"
+                            titleElement="h3"
+                        >
+                            <Alert
+                                isInline
+                                variant="info"
+                                title="Note: the claim mappings are used to map claims returned in underlying identity provider's token to ACS token."
+                            >
+                                <p>
+                                    In case claim mapping is not configured for a certain claim,
+                                    this claim will not be returned from authentication provider.
+                                </p>
+                            </Alert>
+                            {(!values.claimMappings ||
+                                (Array.isArray(values.claimMappings) &&
+                                    values.claimMappings.length === 0)) && (
+                                <p>No claim mappings defined</p>
+                            )}
+                            <FieldArray
+                                name="claimMappings"
+                                render={(arrayHelpers) => (
+                                    <>
+                                        {values.claimMappings &&
+                                            Array.isArray(values.claimMappings) &&
```

Comment:
In that case, `Add claim mapping` button that uses `arrayHelpers` won't show up if there are no claim mappings already. Alternative would be to duplicate `Add claim mapping` code, but that won't be unfortunate.

I checked and `Required attributes` section uses the same approach as the one I used here.
`Rules` section wraps everything in `FieldArray`, whether there is some array to render already or not

Reply:
Separate observation, not a change request. Similar to comment below, you might be able to write condition as `!Array.isArray(values.claimMappings) || values.claimMappings.length === 0` because `Array.isArray` returns false for `undefined` or `null` nullish values.